bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/editor/targetComponent/details/SiderealNameEditor.scala

```diff
@@ -108,14 +107,14 @@ final class SiderealNameEditor extends TelescopePosEditor with ReentrancyHack {
     tqr.getCoordinates(0) match {
       case pos: WorldCoords =>
         try {
-          t.getRa.setValue(pos.getRA.toString)
-          t.getDec.setValue(pos.getDec.toString)
+          t.setRaString(pos.getRA.toString)
+          t.setDecString(pos.getDec.toString)
```

Comment:
There's a `spt.notifyOfGenericUpdate()` down at line 124 which we shouldn't need now.